model/adapter.py

```diff
@@ -116,7 +137,8 @@ def create_exchange(self, metabolite):
         -------
 
         """
-        extracellular_metabolite = Metabolite(metabolite.id[:-2] + '_e', formula=metabolite.formula, compartment='e')
+        extracellular_metabolite = Metabolite(re.sub('_[cpm]$', '_e', metabolite.id),
```

Comment:
Models define their own compartments so actually could be any number or abbreviations :/